SPEC.md

```diff
@@ -124,6 +124,25 @@ work with the project (e.g., an automated documentation generator).
 crystal: 0.19.2
 ```
 
+### repository
+
+The URL of the shard's canonical repository (String, recommended).
+
+The URL should be compatible with typical VCS tools without modifications.
+`http`/`https` is preferred over VCS schemes like `git`.
+It is recommended that this URL is publicly available.
+
+Copies of a shard (such as mirros, development forks etc.) should point to the same:
+canonical repository address, even if hosted at different locations.
+
+### homepage
+
+The URL of the shard's homepage (String, recommended).
```

Comment:
I'm not sure the homepage has to be recommended.

Reply:
Sorry, lost track of this PR.

Yeah I guess it doesn't need to be, but I'd still recommend it 😄  And IMO it makes sense: when your shard has a homepage, you would want to make it publicly known anyway.

Reply:
In many cases, the homepage will be obvious from the repository. But I don't think it bothers. Also, some might use a nicer homepage that the repository one. So 👍 on my side.